_replication-plugin/api.md

```diff
@@ -0,0 +1,243 @@
+:--- | :--- |:--- |:--- |
+`leader_alias` |  The name of the leader cluster. | `string` | Yes
+`leader_index` |  The index on the leader cluster that you want to replicate. | `string` | Yes
+`use_roles` |  The roles to use for all subsequent backend replication tasks between the indices. Specify a `leader_cluster_role` and `follower_cluster_role`. See [Map the leader and follower cluster roles]({{site.url}}{{site.baseurl}}/replication-plugin/permissions/#map-the-leader-and-follower-cluster-roles). | `string` | If security plugin is enabled
+
+#### Sample response
+
+```json
+{
+   "acknowledged": true
+}
+```
+
+## Stop replication
+Introduced 1.1
+{: .label .label-purple }
+
+Terminates replication and converts the follower index to a standard index.
+
+#### Request
+
+```json
+POST /_plugins/_replication/<follower-index>/_stop
+```
+
+#### Sample response
+
+```json
+{
+   "acknowledged": true
+}
+```
+
+## Pause replication
+Introduced 1.1
+{: .label .label-purple }
+
+Pauses replication of the leader index. If you don't resume replication after 12 hours, it stops completely and the follower index is converted to a standard index.
+
+#### Request
+
+```json
+PUT /_plugins/_replication/<follower-index>/_pause
+```
+
+#### Sample response
+
+```json
+{
+   "acknowledged": true
+}
+```
+
+## Resume replication
+Introduced 1.1
+{: .label .label-purple }
+
+Resumes replication of the leader index. 
+
+#### Request
+
+```json
+PUT /_plugins/_replication/<follower-index>/_resume
+```
+
+#### Sample response
+
+```json
+{
+   "acknowledged": true
+}
+```
+
+## Get replication status
+Introduced 1.1
+{: .label .label-purple }
+
+Gets the status of index replication. You can use this API to measure replication lag. Run this command from the leader cluster.
+
+#### Request
+
+```json
+GET /_plugins/_replication/<follower-index>/_status
+```
+
+#### Sample response
+
+```json
+{
+   "status":"SYNCING",
+   "reason":"User initiated",
+   "remote_cluster":"remote-cluster",
+   "leader_index":"leader-01",
+   "follower_index":"follower-01",
+   "syncing_details":{
+      "remote_checkpoint": 19,
+      "local_checkpoint": 19,
+      "seq_no": 20
+   }
+}
+```
+
+To include shard replication details in the response, add `&verbose=true`.
+
+## Update settings
+Introduced 1.1
+{: .label .label-purple }
+
+Updates settings on the follower index.
+
+#### Request
+
+```json
+PUT /_plugins/_replication/<follower-index>/_update
+{
+   "settings":{
+      "index.number_of_shards": 4,
+      "index.number_of_replicas": 2
+   }
+}
+```
+
+#### Sample response
+
+```json
+{
+   "acknowledged": true
+}
+```
+
+## Create replication rule
+Introduced 1.1
+{: .label .label-purple }
+
+Automatically starts replication on indices matching a specified pattern. Newly created indices on the remote cluster that match one of the specified patterns will be automatically configured as follower indices. You can also use this API to update existing auto-follow patterns.
```

Comment:
We still want present tense whenever possible, so something like, "If a new index on the leader cluster matches the pattern, OpenSearch automatically creates a follower index and begins replication."